kwok/cloudprovider/cloudprovider.go

```diff
@@ -118,6 +119,14 @@ func (c CloudProvider) Name() string {
 	return "kwok"
 }
 
+func (c CloudProvider) GetSupportedNodeClasses() schema.GroupVersionKind {
```

Comment:
I don't think there's any need for us to have a KwokNodeClass. What were you thinking we would have in here? Is this just a placeholder because this information would be required? Do we need to enforce that this is non-zero length?